src/controllers/movies.js

```diff
@@ -0,0 +1,47 @@
+const { Prisma } = require('@prisma/client')
+const prisma = require('../utils/prisma')
+
+const getMovies = async (req, res) => {
+  const movies = await prisma.movie.findMany()
+  res.status(200).send({ movies: movies })
+}
+const createMovie = async (req, res) => {
+  const { title, runtimeMins } = req.body
+  const movie = await prisma.movie.create({
+    data: {
+      title,
+      runtimeMins
```

Comment:
In order for the tests to pass, didn't you have to input the title "Top Gun" and RunTime 110?